impl/src/bin/cargo-raze.rs

```diff
@@ -53,9 +53,11 @@ Usage:
     cargo raze [--verbose] [--quiet] [--color=<WHEN>] [--dryrun] [--cargo-bin-path=<PATH>] [--output=<PATH>]
     cargo raze <buildprefix> [--verbose] [--quiet] [--color=<WHEN>] [--dryrun] [--cargo-bin-path=<PATH>]
                              [--output=<PATH>]
+    cargo raze --version
 
 Options:
     -h, --help                         Print this message
+    --version                          Print version info and exit
```

Comment:
can you add in a -V option as well, since both rustc/cargo allow it?

Reply:
Sure. Done.